strands_navigation_msgs/msg/TopologicalNode.msg

```diff
@@ -1,6 +1,8 @@
 string name
 string map
 string pointset
-geometry_msgs/Pose pose
+geometry_msgs/Pose[] pose
```

Comment:
Yes, that is to add the possibility of going to alternative waypoints in the same node if the exact waypoint is blocked, this one of the alternatives we are looking into.